lib/index.js

```diff
@@ -141,6 +143,30 @@ document.querySelector(DOMstrings.favoritesList).addEventListener('click', funct
   }
 })
 
+document.querySelector(DOMstrings.addPlaylistBtn).addEventListener('click', function(event) {
+  let newPlaylistTitle = document.querySelector(DOMstrings.addPlaylistForm).value;
+  if (newPlaylistTitle) {
```

Comment:
I like that you have this `if` statement here. I think a nice future touch would be to have a matching `else` that basically serves as an error catch, and it could display a message to the effect of "You must enter a name for the new playlist!"